app/controllers/users_controller.rb

```diff
@@ -1,8 +1,7 @@
 class UsersController < ApplicationController
   def show
-    # @user = User.find_by(params[:user_id])
-    # @user = User.find_by(@current_user)
-    @user = User.find_by(id: session[:user_id])
+    @user = current_user
+    @bookmarks = current_user.videos
```

Comment:
Talked to Jack C and he said he couldnt figure out how to do this in active record either. I am using basic ruby instead now.